plugins/Deflect/DeflectPixelOp.cpp

```diff
@@ -19,16 +19,18 @@
  */
 
 #include "DeflectPixelOp.h"
+#include "utils.h"
 
 #include <ospray/SDK/fb/FrameBuffer.h>
 
+#include <engines/ospray/utils.h>
+
 namespace
 {
-template <typename T>
-std::future<T> make_ready_future(const T value)
+std::future<bool> make_ready_future()
 {
-    std::promise<T> promise;
-    promise.set_value(value);
+    std::promise<bool> promise;
+    promise.set_value(true);
     return promise.get_future();
 }
```

Comment:
I remember mentioning in another PR that this function was not needed, did you do any changes back then?

Reply:
the function is needed, no? I don't remember what the solution for the comment the other day was.